### ISSUE TITLE
Search in the editorial notes

### ISSUE THREAD

**Issue by ptrckb** [pkp/pkp-lib#5307]
The user biography and review interests are considered when searching for a reviewer with a phrase but not the editorial notes/gossip.

**Comment by ptrckb** [comment]
pkp/pkp-lib#5308 '(`stable-3_1_2`)
pkp/ojs#2542 (`stable-3_1_2`)

Would also be relevant for `master`.

**Comment by asmecher** [comment]
Good idea, @ptrckb! Would you be able to sign the CLA (https://github.com/pkp/ojs/pull/2542#issuecomment-559518391)?

**Comment by NateWr** [comment]
I don't think we can include this because it will leak private information. The editorial notes have some [access restrictions](https://github.com/pkp/pkp-lib/blob/master/classes/services/PKPUserService.inc.php#L424-L457) including things like not allowing a user to access their own gossip fields.

If we can search the gossip fields, I could reverse engineer my own gossip by searching for my name + words used in the gossip field.

**Comment by asmecher** [comment]
Closing as out of date.
